tests/test_matrix_utils_connect.py

```
@@ -4,14 +4,40 @@
+from typing import Any
+from unittest.mock import AsyncMock, MagicMock, Mock, patch
 
 import pytest
 
 from mmrelay.matrix_utils import MissingMatrixRoomsError, connect_matrix
 
 
+class _ImmediateAwaitable:
+    """Awaitable that resolves immediately without creating coroutine objects."""
+
+    def __init__(self, value: Any = None) -> None:
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)